packages/evm-adapter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@heliofi/evm-adapter",
-  "version": "0.0.10-beta.0",
+  "version": "0.0.18-beta",
```

Comment:
why do you tag it beta?

Reply:
It collides with tags from other packages in project for some reason